app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -922,16 +921,18 @@ class PipelineSampleReads extends React.Component {
                 onClick: () => window.open(`/samples/${sampleId}`, "_self")
               }))}
             />
-            {this.props.admin && (
-              <div className={cs.sampleDetailsLinkContainer}>
-                <span
-                  className={cs.sampleDetailsLink}
-                  onClick={this.toggleSampleDetailsSidebar}
-                >
-                  Sample Details
-                </span>
-              </div>
-            )}
+            {this.pipelineRun &&
+              this.pipelineRun.finalized === 1 &&
```

Comment:
Please use a condition based on `reportReady` here. You can fix `reportReady` by changing https://github.com/chanzuckerberg/idseq-web/blob/255a906df80052050dfebbfe21a68809750d53fa/app/controllers/samples_controller.rb#L279 to `if @pipeline_run.report_ready?` (we may need to work through some edge cases together). You can fix the `report_ready?` function by editing https://github.com/chanzuckerberg/idseq-web/blob/master/app/models/pipeline_run.rb#L105 -- should be `REPORT_READY_OUTPUT = "contig_counts".freeze` or perhaps `REPORT_READY_OUTPUT = "taxon_byteranges".freeze` to be sure. Let's discuss with @yunfangjuan